src/backend/columnar/sql/udfs/citus_finish_pg_upgrade/10.0-1.sql

```diff
@@ -0,0 +1,26 @@
+IF version() ~ '12' or version() ~ '13' THEN
```

Comment:
these should also be fixed 

note that `current_setting('server_version')` gives a simpler version number